src/gfnff/gfnff_setup.f90

```diff
@@ -62,14 +62,14 @@ subroutine gfnff_setup(env,verbose,restart,mol,p_ext_gfnff,gen,param,topo,accura
   if (restart) then
      inquire(file='gfnff_topo', exist=ex)
      if (ex) then
-       call read_restart_gff('gfnff_topo',mol%n,p_ext_gfnff,success,.true.,topo)
+       call read_restart_gff(env,'gfnff_topo',mol%n,version,success,.true.,topo)
        !hbrefgeo is usually set within gfnff_ini2/gfnff_hbset0 equal to initial xyz
        topo%hbrefgeo=mol%xyz
        if (success) then
           write(env%unit,'(10x,"GFN-FF topology read from file successfully!")')
           return
        else
-          call env%warning("Could not read topology file", source)
+          call env%warning("Could not read topology file, generated new one", source)
           call env%check(exitRun)
           if (exitRun) then
              return
```

Comment:
Not necessarily, in case of `--strict` this will turn into an error in the backtrace and the added comment will be confusing.

Reply:
done